translations/readme/README-en.md

```diff
@@ -0,0 +1,16 @@
+# QCTF School 2018
+
+This is the repository for the development of QCTF School 2018 competitions - full-time CTF competitions for schoolchildren held by the team [HackerDom](http://hackerdom.ru). The competition was held on May 13, 2018 at the [26 venues](http://qctf.ru/school-2018/places).
+
+The rules of QCTF School can be found on the official website of the competition — http://qctf.ru/school-2018/rules
```

Comment:
+ `(in russian)`

Reply:
you need the url text ( [HackerDom] and [26 venues] ) to be in russian ?

Reply:
No, I just wanted to add clarification into text that link content is in russian

Reply:
Sorry for the confusion